### PR TITLE
Feat/setup sentry

### PR DESCRIPTION
**Pull request by duke-eth**

### CODE REVIEW

**Review by nonsensecodes — APPROVED** [review]
Thank you for setting this up, @duke-eth!

File: .env.qa [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NODE_ENV=development
+NODE_ENV=qa
```
Comment (amaredeus) [inline .env.qa:1]: @nonsensecodes @duke-eth This breaks logging in Log.ts, why was this changed?

File: src/app/utils/SentryUtils.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+		const transaction = Sentry.startTransaction({
+			op: 'command',
+			name: ctx.commandName,
+		});
+
+		Sentry.configureScope(scope => {
+			scope.setSpan(transaction);
+			scope.setUser({
+				id: ctx.member.user.id,
```
Comment (amaredeus) [inline src/app/utils/SentryUtils.ts:18]: @duke-eth What happens for commands that can be executed in DMs? 

As I'm enabling strict mode, it's shown that member can come back as null because the context of the execution can happen inside or outside of a guild
